### PR TITLE
[Feature Request] - Download Complete Remidner #3475

### LINKED ISSUE #3475: [Feature Request] Download Complete Reminder
Showing a reminder/sign that the download is completed

### PR DESCRIPTION
**Pull request by moe-attack**
After seeing the request I think it is a really good feature to have so I made some minor updates to get it running.

Show a download complete notification channel when all downloads are completed. Auto cancels when onclick and navigate to download screen.

Closes #3475

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -671,6 +671,7 @@
     <string name="download_notifier_text_only_wifi">No Wi-Fi connection available</string>
     <string name="download_notifier_no_network">No network connection available</string>
     <string name="download_notifier_download_paused">Download paused</string>
+    <string name="download_notifier_download_finish">The download has been completed</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml:674]: Maybe shorten this to just "Download complete"?

Comment (moe-attack) [reply app/src/main/res/values/strings.xml]: Easy done! That being said, I made the string "Download completed" because it should be past tense.

**Comment by arkon** [comment]
Thanks for the contribution!

**Comment by moe-attack** [comment]
I'm a weeb myself so I'd love to contribute to this kind of app :D I'll help out again soon!

**Comment by Kiyazami** [comment]
Thank you so much for fulfilling my request. I really need this
